libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/CohortStatsHeatmap.tsx

```diff
@@ -115,7 +96,7 @@ export class CohortStatsHeatmap extends React.Component<
                 // make metric name lower case in sentence
                 this.series.xAxis.categories[this.point.x].toLowerCase(),
                 this.series.yAxis.categories[this.point.y],
-                pointValue
+                pointValue === null ? "N/A" : pointValue

```

Comment:
Is this visiable to user? Probably need localize

Reply:
Done